package.json

```diff
@@ -11,7 +11,8 @@
         "flow": "flow",
         "flow.status": "flow status",
         "flow.check": "flow check",
-        "flow.coverage.html": "flow-coverage-report -i \"src/**/*.js\" -t html"
```

Comment:
First of all, checkout from `develop` branch and in PR set `develop` as base. Because in develop I refactored script section few weeks ago.

Reply:
Also, we work in only on `develop`. I add this rule at [Contributing section in README](https://github.com/moira-alert/web2.0/blob/develop/README.md#contributing)